docs/accelerometer.rst

```diff
@@ -8,31 +8,36 @@ also provides convenience functions for detecting gestures. The
 recognised gestures are: ``up``, ``down``, ``left``, ``right``, ``face up``,
 ``face down``, ``freefall``, ``3g``, ``6g``, ``8g``, ``shake``.
 
+By default MicroPython sets the accelerometer to +/- 2g, and this can only be
+modified in the DAL layer below. The accelerometer returns a value in the range 
```

Comment:
We haven't really introduced the concept of the DAL in these docs, so we should probably rephrase it to something along the lines of:
`and changing this functionality is not currently exposed in MicroPython.`